source/common/http/filter_manager.cc

```diff
@@ -919,7 +936,7 @@ FilterManager::commonEncodePrefix(ActiveStreamEncoderFilter* filter, bool end_st
         state_.decoder_filter_chain_aborted_ = true;
       }
     }
-    return encoder_filters_.begin();
+    return encoder_filters_.rbegin();
```

Comment:
why are we changing these to reverse iterators?
/wait-any

Reply:
I think it is commented at the function where the filter is inserted to the container. I will add some more details at there.